kubernetes/AKS/README.md

```diff
@@ -0,0 +1,262 @@
+## TorchServe on Azure Kubernetes Service (AKS)
+
+### 1 Create an AKS cluster
+
+This quickstart requires that you are running the Azure CLI version 2.0.64 or later. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli).
+
+#### 1.1 Set Azure account information
+
+```az login``` 
+
+```az account set -s your-subscription-ID```
+
+#### 1.2 Create a resource group
+
+An Azure resource group is a logical group in which Azure resources are deployed and managed. When you create a resource group, you are asked to specify a location. This location is where resource group metadata is stored, it is also where your resources run in Azure if you don't specify another region during resource creation. Create a resource group using the [az group create](https://docs.microsoft.com/en-us/cli/azure/group#az-group-create) command.
+
+The following example creates a resource group named *myResourceGroup* in the *eastus* location.
+
+```az group create --name myResourceGroup --location eastus```
+
+#### 1.3 Create AKS cluster
+
+Use the [az aks create](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create) command to create an AKS cluster. The following example creates a cluster named *myAKSCluster* with one node. This will take several minutes to complete.
+
+```az aks create  --resource-group myResourceGroup  --name myAKSCluster --node-vm-size Standard_NC6   --node-count 1```
+
+#### 1.4 Connect to the cluster
+
+To manage a Kubernetes cluster, you use [kubectl](https://kubernetes.io/docs/user-guide/kubectl/), the Kubernetes command-line client. If you use Azure Cloud Shell, `kubectl` is already installed. To install `kubectl` locally, use the [az aks install-cli](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-install-cli) command:
+
+```az aks install-cli```
+
+To configure `kubectl` to connect to your Kubernetes cluster, use the [az aks get-credentials](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-get-credentials) command. This command downloads credentials and configures the Kubernetes CLI to use them.
+
+```az aks get-credentials --resource-group myResourceGroup --name myAKSCluster```
+
+#### 1.5 Install NVIDIA device plugin
+
+Before the GPUs in the nodes can be used, you must deploy a DaemonSet for the NVIDIA device plugin. This DaemonSet runs a pod on each node to provide the required drivers for the GPUs.
+
+```kubectl apply -f nvidia-device-plugin-ds.yaml```
```

Comment:
We don't have the .yaml file yet in this step because we clone the repo in step 2.1. Can we change the order so that the repo is cloned earlier or the NVidia plugin is applied later?

Reply:
Thanks for your comments. The order of AKS/README.md has been changed.